common/build/eslint-config-fluid/scripts/cleanup-printed-configs.js

```diff
@@ -24,6 +24,6 @@ const sortJson = require("sort-json");
 		fs.writeFileSync(filePath, JSON.stringify(json, undefined, 4));
 
 		// Sort the JSON in-place.
-		sortJson.overwrite(filePath, { indentSize: 4 });
+		sortJson.overwrite(filePath, { indentSize: 4, depth: 2 });
```

Comment:
It's unfortunate that the key order is significant for import/resolver, because the sorting really helps reviews. My preference is to keep the sort as it is, at least for this PR. If you feel strongly about it, I can open a PR with just the depth change and merge it first.

Reply:
Split into #20862.

IMO if sorting at depth 2 is still disruptive to reviews, it's worth just calling this fact out where we specify import/resolver in config. It seems to me like for the most part, key order of individual rule configs won't change that much over time, which is the main thing that's not sorted.